# Request fields
int8   interaction_id
string bag_file
---
# Response fields
Statistics stats
